coopersystem-fsd/pcc-framework
# Import user via CSV file

## Description

- Adds a submenu in the Users menu for importing users via CSV
- The import page has a field for importing the file and a tab for configuring the messages that the user will receive
- Event listing page now shows event ID
- User events can be edited manually in user profile and user will be notified of new event
- In the event editor there are new fields for Open Collective information to generate the donation/contribution button and with the option to show content before the button

## Steps to test

1. Go to Admin > Users > Import users CSV
2. Choose a CSV file that is within the parameters mentioned in the instructions and click on Load File

**Expected behavior: A log will be displayed after the file has been loaded and the emails sent
